bin/inference/pycbc_inference_start_from_samples

```diff
@@ -0,0 +1,49 @@
+#!/bin/env python
+""" Convert inference file to parallel temperred compatible start format
+"""
+
+import argparse
+import numpy
+import h5py
+from numpy.random import choice
+from pycbc.inference.io import loadfile
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--input-file')
+parser.add_argument('--output-file')
+parser.add_argument('--sampler', default='emcee_pt_file',
+    help="The output sampler type, if none, we assume emcee_pt")
+parser.add_argument('--ntemps', type=int)
+parser.add_argument('--nwalkers', type=int)
+args = parser.parse_args()
+
+# populate an emcee start file with
+# values chosen from a dynesty file
+# each temperature and walker will get a random
+# point from the dynesty output
+
+ntemps = args.ntemps
+nwalkers = args.nwalkers
+
+f = loadfile(args.input_file)
+params = list(f.variable_params) + ['loglikelihood']
+samples = f.read_samples(params)
+nsample = len(samples)
+
+# These are the ids we'll use for the temps / walkers
+use = choice(nsample, replace=False, size=ntemps * nwalkers)
+
+o = loadfile(args.output_file, 'w', filetype=args.sampler)
```

Comment:
Get the filetype using the sampler class:
```
filetype = samplers[args.sampler].io.name
```
(You'll need to add a `from pycbc.inference.sampler import samplers` for that to work)